docs/reference/readme.md

```diff
@@ -185,6 +185,19 @@ Allowed domains is a collection of whitelisted domains to authorize for a given
 
 Allow unauthenticated HTTP OPTIONS requests as [per the CORS spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Preflighted_requests).
 
+#### Public Access
+
+- `yaml`/`json` setting: `allow_public_unauthenticated_access`
```

Comment:
Let me know if this name is OK, or if you'd prefer something along the lines of `disable_all_access_control`

Reply:
I can't think of anything better. `disable_all_access_control` **is** descriptive. 

I'll try to see what other products call their "turn-off-all-access-control" flags just to see but I think this is fine.

Reply:
You pick, I'm open to either. 👍